0,NameError,name 'unemplotment' is not defined,0,,2023-05-22 16:34:05.974135,"[('by_nei_pter = unemplotment.sort(""NEI-PTER"", descending=True)', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/2626414609.py"", line 2, in <module>', '    by_nei_pter = unemplotment.sort(""NEI-PTER"", descending=True)', ""NameError: name 'unemplotment' is not defined""]"
1,AttributeError,'Table' object has no attribute 'colummn',0,,2023-05-22 16:42:24.139359,"[('pter = unemployment.colummn(""NEI-PTER"") - unemployment.column(""NEI"")', 1)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/3580100085.py"", line 1, in <module>', '    pter = unemployment.colummn(""NEI-PTER"") - unemployment.column(""NEI"")', ""AttributeError: 'Table' object has no attribute 'colummn'""]"
2,SyntaxError,"invalid syntax (3981950045.py, line 4)",0,,2023-05-22 16:50:24.437032,"[('return compile(source, filename, symbol, self.flags | PyCF_ONLY_AST, 1)', 86)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3241, in run_cell_async', '    code_ast = compiler.ast_parse(cell, filename=cell_name)', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/compilerop.py"", line 86, in ast_parse', '    return compile(source, filename, symbol, self.flags | PyCF_ONLY_AST, 1)', '  File ""/tmp/ipykernel_10470/3981950045.py"", line 4', '    ""Year"", year', '    ^^^^^^', 'SyntaxError: invalid syntax']"
3,TypeError,Table.with_column() missing 1 required positional argument: 'values',0,,2023-05-22 17:40:36.525429,"[('migration_rates = Table.with_column(""Migration Rate"", abs(pop.column(""MIGRATION"")/pop.column(""2015"")))', 1)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/1469160995.py"", line 1, in <module>', '    migration_rates = Table.with_column(""Migration Rate"", abs(pop.column(""MIGRATION"")/pop.column(""2015"")))', ""TypeError: Table.with_column() missing 1 required positional argument: 'values'""]"
4,NameError,name 'count' is not defined,0,,2023-05-22 17:40:50.536130,"[('movers = count(migration_rates.where(""Migration Rate"", are_above(0.01)))', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/2911554400.py"", line 2, in <module>', '    movers = count(migration_rates.where(""Migration Rate"", are_above(0.01)))', ""NameError: name 'count' is not defined""]"
5,AttributeError,module 'numpy' has no attribute 'count',0,,2023-05-22 17:41:46.407795,"[('movers = np.count(migration_rates.where(""Migration Rate"", are_above(0.01)))', 2), ('raise AttributeError(""module {!r} has no attribute ""', 311)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/2062950977.py"", line 2, in <module>', '    movers = np.count(migration_rates.where(""Migration Rate"", are_above(0.01)))', '  File ""/opt/conda/lib/python3.10/site-packages/numpy/__init__.py"", line 311, in __getattr__', '    raise AttributeError(""module {!r} has no attribute ""', ""AttributeError: module 'numpy' has no attribute 'count'""]"
6,NameError,name 'are_above' is not defined,0,,2023-05-22 17:43:24.141418,"[('movers = migration_rates.where(""Migration Rate"", are_above(0.01))', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/3269263736.py"", line 2, in <module>', '    movers = migration_rates.where(""Migration Rate"", are_above(0.01))', ""NameError: name 'are_above' is not defined""]"
7,TypeError,Table.column() missing 1 required positional argument: 'index_or_label',0,,2023-05-22 17:44:21.617309,"[('movers = migration_rates.where(""Migration Rate"", are.above(0.01)).column()', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/972758693.py"", line 2, in <module>', '    movers = migration_rates.where(""Migration Rate"", are.above(0.01)).column()', ""TypeError: Table.column() missing 1 required positional argument: 'index_or_label'""]"
8,NameError,name 'count' is not defined,0,,2023-05-22 17:45:35.531473,"[('movers = count(migration_rates.where(""Migration Rate"", are.above(0.01)).column(""Migration Rate""))', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/3505760143.py"", line 2, in <module>', '    movers = count(migration_rates.where(""Migration Rate"", are.above(0.01)).column(""Migration Rate""))', ""NameError: name 'count' is not defined""]"
9,SyntaxError,"incomplete input (2237204549.py, line 4)",0,,2023-05-22 18:18:37.019585,"[('return compile(source, filename, symbol, self.flags | PyCF_ONLY_AST, 1)', 86)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3241, in run_cell_async', '    code_ast = compiler.ast_parse(cell, filename=cell_name)', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/compilerop.py"", line 86, in ast_parse', '    return compile(source, filename, symbol, self.flags | PyCF_ONLY_AST, 1)', '  File ""/tmp/ipykernel_10470/2237204549.py"", line 4', '    Table().with_columns(""Birth rates"", birth_rates, ""Death rates"", death_rates).scatter((""Birth rates"", ""Death rates"")', '                                                                                                                       ^', 'SyntaxError: incomplete input']"
10,NameError,name 'manilla' is not defined,0,,2023-05-22 18:54:14.859094,"[('manilla.hist(""ride time"", bins=equal_bins)', 2)]","['Traceback (most recent call last):', '  File ""/opt/conda/lib/python3.10/site-packages/IPython/core/interactiveshell.py"", line 3508, in run_code', '    exec(code_obj, self.user_global_ns, self.user_ns)', '  File ""/tmp/ipykernel_10470/603609541.py"", line 2, in <module>', '    manilla.hist(""ride time"", bins=equal_bins)', ""NameError: name 'manilla' is not defined""]"
